numerrin-interface/libpcore.cpp

```diff
@@ -0,0 +1,1041 @@
+#include <sstream>
```

Comment:
Instructions are missing from Readme.rst  (i.e. there is no mention of `.bashrc` or how/where to install libnumerrin4.so and numerrin.so).